exporter/datadogexporter/translate_traces.go

```diff
@@ -381,6 +382,10 @@ func setStringTag(s *pb.Span, key, v string) {
 			setMetric(s, ext.EventSampleRate, 0)
 		}
 	default:
+		if len(v) > MaxMetaValLen {
+			v = utils.TruncateUTF8(v, MaxMetaValLen) + "..."
+		}
+
```

Comment:
Why do we want to do this only when it's not a service name/span type/analytics event? Some of those can have arbitrary values, right? Why not do this every time?

Reply:
yea, idk, i rushed things. should be for everything, i'll update